jwst/extract_1d/psf_profile.py

```diff
@@ -129,37 +131,84 @@ def _make_cutout_profile(xidx, yidx, psf_subpix, psf_data, dispaxis,
     return [sprofile, nod_profile * -1]
 
 
-def _profile_residual(shifts_to_optimize, cutout, cutout_var, xidx, yidx,
-                      psf_subpix, psf_data, dispaxis, fit_bkg=True):
-    """Residual function to minimize for optimizing trace locations."""
+def _profile_residual(
+    shifts_to_optimize, cutout, cutout_var, xidx, yidx, psf_subpix, psf_data, dispaxis, fit_bkg=True
+):
+    """
+    Residual function to minimize for optimizing trace locations.
+
+    Call `_make_cutout_profile` to generate a profile from input parameters.
+    Call `extract1d` to generate a scene model from the data and the new profile.
+    Compute a residual value from the sum of (model - cutout) ** 2 / cutout_var.
+
+    Parameters
+    ----------
+    shifts_to_optimize : list of float
+        The first value is used as the `extra_shift` parameter to
+        `_make_cutout_profile`.  If two are provided, the second value is
+        used as the `nod_offset` parameter to `_make_cutout_profile`.
+        Otherwise, no nod offset is applied.
```

Comment:
```suggestion
        `_make_cutout_profile`.  The second value, if provided, is
        used as the `nod_offset` parameter to `_make_cutout_profile`; 
        otherwise, no nod offset is applied.
```
As written it was initially unclear to me what "Otherwise" referred to.  Not sure my suggestion is perfect, but it does clarify that the otherwise goes with "if second value provided"